modules/languages/python.nix

```diff
@@ -36,6 +36,69 @@ with builtins; let
       '';
     };
   };
+
+  defaultDebugger = "debugpy";
+  debuggers = {
+    debugpy = {
+      # idk if this is the best way to install/run debugpy
+      package = pkgs.python3.withPackages (ps: with ps; [debugpy]);
```

Comment:
is there a reason we provide python3 alongside the debugpy package? [python311Packages.debugpy](https://search.nixos.org/packages?channel=23.05&show=python311Packages.debugpy&from=0&size=50&sort=relevance&type=packages&query=debugpy) is already packaged for nixpkgs

Reply:
I simply don't know of a better way. the debugpy package provides the module but its not executable, you need a python binary to use it: `python -m debugpy`. I could in theory add debugpy to PYTHON_PATH and the user can use whatever python binary they have, but, too complicated.

Reply:
Fair point, let's keep it as is for the time being.